enterprise/frontend/src/metabase-enterprise/sandboxes/components/EditSandboxingModal/EditSandboxingModal.tsx

```diff
@@ -320,9 +319,8 @@ const TargetName = ({ policy, target }: TargetNameProps) => {
       const fieldRef = target[1];
 
       return (
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
         <QuestionLoader
+          questionHash={undefined}
```

Comment:
This error is given incorrectly (due to `@types/prop-types` package) and can be fixed by refactoring `QuestionPicker` to TS.

Reply:
ignored